crates/bevy_camera/src/camera.rs

```diff
@@ -188,11 +169,11 @@ pub struct RenderTargetInfo {
 /// Holds internally computed [`Camera`] values.
 #[derive(Default, Debug, Clone)]
 pub struct ComputedCameraValues {
-    clip_from_view: Mat4,
-    target_info: Option<RenderTargetInfo>,
+    pub clip_from_view: Mat4,
+    pub target_info: Option<RenderTargetInfo>,
     // size of the `Viewport`
-    old_viewport_size: Option<UVec2>,
-    old_sub_camera_view: Option<SubCameraView>,
+    pub old_viewport_size: Option<UVec2>,
+    pub old_sub_camera_view: Option<SubCameraView>,
```

Comment:
making this pub was necessary because bevy_render needs to read/write them

Reply:
`ComputedCameraValues` cannot live in `bevy_render` because it is a member of `Camera`, and `bevy_camera` cannot write these values because `camera_system` needs access to `ManualTextureViews` which hold `TextureView` which are render resources. This can theoretically be extracted into bevy_image by manually expanding `define_atomic_id!`, but i'd rather leave that for a follow-up PR as this one is already far too big.